backend/src/main/resources/application.yaml

```diff
@@ -11,4 +11,16 @@ spring:
         default_schema: simple_report
   liquibase:
     default-schema: ${spring.jpa.properties.hibernate.default_schema}
+graphql:
+  servlet:
+    mapping: /graphql
+    enabled: true
+    corsEnabled: true
+  tools:
+    schema-location-pattern: "**/*.graphqls"
+okta:
+  oauth2:
+    issuer: https://prime-eval.okta.com/oauth2/default
+    client-id: ${clientID}
```

Comment:
This is probably not the best way to pass this information in, but there are a bunch of options. We can discuss what fits best.